README.md

```diff
@@ -1,44 +1,39 @@
 #Android Streaming Client
 
-##About
+##Introduction
 
-Android Streaming Client is an Android library which main purpose is to play real <br>
+###What it does?
+
+**Android Streaming Client** is an Android library which main purpose is to play real <br>
 time video in an Android device. <br>
 
+###How it does it
+
 The current version of the code only supports RTP over UDP as the transport <br>
 protocol and decodes H264 encoded video. <br>
-It currently only works with video streamed from [libstreaming library](https://github.com/fyhertz/libstreaming) <br>
-running in a separate Android device. We are using a custom version of <br>
-libstreaming which is the original library plus some minimal changes made to <br>
-fix some issues we encountered while we were working on the Android Streaming <br>
-Client library.<br>
-
-##Content of the project
+It uses [efflux library](https://github.com/brunodecarvalho/efflux) to create an underlying RTP session to listen to <br>
+package arrival. <br>
+It only works with video streamed from [libstreaming library](https://github.com/fyhertz/libstreaming) running in a <br>
```

Comment:
A small suggestion here: why not instead say that we have only tested so far with this client (as opposed to: it only works with this client).
I would also personally demote the "player" part of this to much later in the readme. I would make the readme much more about the library itself and how to use it, and put some information later about which players we have tested with and how.
This is only a suggestion.